pajbot/apiwrappers/twitch/tmi.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from pajbot.apiwrappers.twitch.base import BaseTwitchAPI
+
+log = logging.getLogger(__name__)
+
+
+class TwitchTMIAPI(BaseTwitchAPI):
+    authorization_header_prefix = "OAuth"
+
+    def __init__(self, client_credentials, redis):
+        super().__init__(base_url="https://tmi.twitch.tv/", redis=redis)
+        self.session.headers["Accept"] = "application/vnd.twitchtv.v5+json"
+        self.client_credentials = client_credentials
+
+    @property
+    def default_authorization(self):
+        return self.client_credentials
+
+    def get_chatters(self, streamer):
```

Comment:
This should be called `fetch_chatters` since it fetches the chatters. If you want to provide a cache layer, add a `get_chatters` function that uses `self.cache.cache_fetch_fn`.